examples/llamacheck/llamacheck.cpp

```diff
@@ -0,0 +1,194 @@
+#include "common.h"
+#include "llama.h"
+
+#include <cmath>
+#include <cstdio>
+#include <string>
+#include <vector>
+
+int main(int argc, char ** argv) {
+    gpt_params params;
+
+    if (argc == 1 || argv[1][0] == '-') {
+        printf("usage: %s MODEL_PATH [PROMPT]\n" , argv[0]);
+        return 1 ;
+    }
+
+    if (argc >= 2) {
+        params.model = argv[1];
+    }
+
+
```

Comment:
Could you fix whitespaces in this file? There are multiple instances of double new lines and new lines after comments that shouldn't be there.